ui/apps/platform/src/Containers/Collections/RuleSelector/ByLabelSelector.tsx

```diff
@@ -1,59 +1,63 @@
 import React from 'react';
-import {
-    Flex,
-    Label,
-    FormGroup,
-    FlexItem,
-    Button,
-    Divider,
-    ValidatedOptions,
-    TextInput,
-} from '@patternfly/react-core';
+import { Label, Button, Divider, ValidatedOptions } from '@patternfly/react-core';
 import { TrashIcon } from '@patternfly/react-icons';
 import { FormikErrors } from 'formik';
 import cloneDeep from 'lodash/cloneDeep';
 
 import useIndexKey from 'hooks/useIndexKey';
-import { SelectorEntityType, ScopedResourceSelector, ByLabelResourceSelector } from '../types';
+import {
```

Comment:
Small request to sort these import names, because so many of the others are sorted.